### PR TITLE
Vet center - main page cc content sections

### PR DESCRIPTION
**Pull request by brangi**
## Description

Resolves centralized content stories 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/22414
https://github.com/department-of-veterans-affairs/va.gov-team/issues/22392
https://github.com/department-of-veterans-affairs/va.gov-team/issues/22397

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/vetCenter.graphql.js [hunk 1]
```diff
@@ -57,6 +99,11 @@ const vetCenterFragment = `
         fieldHealthServices {
           entity {
             ... on NodeVetCenterFacilityHealthServi {
+             fieldBody {
+                value
+                format
```
Comment (mpelzsherman) [inline src/site/stages/build/drupal/graphql/vetCenter.graphql.js:104]: Do we need `value` and `format`?

**Review by mpelzsherman — APPROVED** [review]
LGTM aside from the 2 extra fields in the query.

We're still blocked on the centralized q & a content, right?

File: src/site/stages/build/drupal/graphql/vetCenter.graphql.js (hunk 1 above)
Comment (brangi) [reply src/site/stages/build/drupal/graphql/vetCenter.graphql.js]: no really, removed

**Comment by brangi** [comment]
> LGTM aside from the 2 extra fields in the query.
> 
> We're still blocked on the centralized q & a content, right?

yes, still blocked, will start moving those stories to blocked

**Pull request merged**